src/WPF/ArcGISRuntime.WPF.Samples/Samples/Hydrography/EncDisplaySettings/EncDisplaySettings.xaml.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Hydrography;
+using Esri.ArcGISRuntime.Mapping;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace ArcGISRuntime.WPF.Samples.EncDisplaySettings
```

Comment:
Unfortunately we are going to have to rename this sample. We have problems when the name of a sample is the same an and ESRI type, which in this case also happens to be: Esri.ArcGISRuntime.Hydrography.EncDisplaySettings. I would recommend: Change ENC display settings (name on disk --> ChangeEncDisplaySettings).